.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Bug report
+about: Create a report to help us improve EBS CSI Driver
```

Comment:
Please correct the text to point to powervs driver

Reply:
fixed